### PR TITLE
Document trackingData search endpoint

### PR DESCRIPTION
**Pull request by andressa**
This commit adds documentation around trackingData search mechanism. It explains:
- How to do filtering
- How to do aggregation
- How to enrich your trackingData payload.

### CODE REVIEW

File: source/includes/telemetry_api/_trackingData.md [hunk 1]
```diff
@@ -64,6 +64,66 @@ When the information is returned with success, a list with one or more tracking
...
+    --header "Authorization: Bearer {YOUR ACCESS TOKEN}" \
+    "https://agco-fuse-trackers-sandbox.herokuapp.com/trackingData/search?links.canVariable=ENGINE_HOURS"
+```
+
+### Aggregating information
+
+In case you want to group equipment information by its id, you can add a
+combination of 2 fields, as shown in the example:
+
+  - `aggregations`, which should be equal to `equipmentAgg`
```
Comment (bltavares) [inline source/includes/telemetry_api/_trackingData.md]: I didn't quite understand this part. Is the `aggregations` property only possible to be equal to `equipmentAgg`? Is there other possibilities for the fields?

Comment (waldemarnt) [inline source/includes/telemetry_api/_trackingData.md]: I think that could be more clear about aggregation names like  "equipmentAgg". This is the given name of the created aggregation, this name can be anything that you want.

File: source/includes/telemetry_api/_trackingData.md [hunk 2]
```diff
@@ -64,6 +64,69 @@ When the information is returned with success, a list with one or more tracking
...
+In case you want to group information by properties, you can create and
+use aggregations. To create a new aggregation, you need to add a field
+`aggregations` and in this case, it is up to you to provide an
+aggregation name, as its value. In our  exemple, the created aggregation
+name is `equipmentAgg`. The result of this aggregation is going to be
+included in the `meta` index, as a list of created aggregations.
+
+You can also sort or limit the aggregation properties. You can find
+further information about aggregations operators
+[here](https://github.com/agco/agco-json-api-profiles/blob/master/public/search-profile.md).
```
Comment (waldemarnt) [inline source/includes/telemetry_api/_trackingData.md]: 👍

**Comment by waldemarnt** [comment]
👍

**Comment by bltavares** [comment]
I think the documentation would be richer if we could inform what are the possible information we can aggregate on the `/trackingdata` call.

As it is currently, I'm being informed that I could enrich the responses with more information. I see on the example that we can include information from CAN variables, but I kept wondering which other information can be added to the data model.

**Comment by glauco** [comment]
@bltavares we updated the documentation to be more descriptive on that aspect.
Does it look better now?

**Comment by bltavares** [comment]
LGTM

**Pull request merged**